Templating/LazyFilterExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Templating;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class LazyFilterExtension extends AbstractExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('imagine_filter', [FilterRuntime::class, 'filter']),
+        ];
+    }
+}
```

Comment:
I think we are missing

```php
    /**
     * {@inheritdoc}
     */
    public function getName()
    {
        return 'liip_imagine';
    }
```

or?

Reply:
Missed the fact that Twig 1 was still supported :/